main/asciidoc/APKBUILD

```diff
@@ -1,44 +1,40 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=asciidoc
 pkgver=8.6.9
-pkgrel=2
+pkgrel=3
 pkgdesc="Text based documentation"
-url="http://www.methods.co.nz/asciidoc/"
+url="http://asciidoc.org/"
 arch="noarch"
 license="GPL2"
 depends="python2 libxml2-utils docbook-xsl"
-makedepends=""
-install=""
+makedepends="autoconf"
 subpackages="$pkgname-doc $pkgname-vim::noarch"
-source="http://sourceforge.net/projects/asciidoc/files/asciidoc/$pkgver/asciidoc-$pkgver.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/asciidoc/asciidoc/archive/$pkgver.tar.gz"
 
-builddir="$srcdir"/$pkgname-$pkgver
+builddir="$srcdir/$pkgname-$pkgver"
 build() {
 	cd "$builddir"
+	autoconf
```

Comment:
Running `autoconf` is usually done in `prepare()`. Care to change this?

Reply:
I fixed it.